extensions/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/OpenTelemetryBuildItem.java

```diff
@@ -1,18 +0,0 @@
-package io.quarkus.opentelemetry.deployment;
-
-import io.opentelemetry.api.OpenTelemetry;
-import io.quarkus.builder.item.SimpleBuildItem;
-import io.quarkus.runtime.RuntimeValue;
-
-public final class OpenTelemetryBuildItem extends SimpleBuildItem {
```

Comment:
I assume this is no longer used by anyone?

Reply:
No, it was dead code needed by the previous implementation of the exporter